### ISSUE TITLE
Indexing problem

### ISSUE THREAD

**Issue by laurensorensen** [sul-dlss/exhibits#2346]
Hi. We tried re-indexing this [test exhibit](https://exhibits.stanford.edu/laurens-vt-test/dashboard) yesterday (twice) and it failed both times. Attaching a screenshot of the error messages. 

First try indexing:
<img width="660" alt="Screenshot 2023-08-23 at 10 08 31 AM" src="https://github.com/sul-dlss/exhibits/assets/3269689/78dc3998-f030-4f0b-9643-0ebd87d78a06">

Second try indexing: 
<img width="675" alt="Screenshot 2023-08-23 at 10 08 59 AM" src="https://github.com/sul-dlss/exhibits/assets/3269689/f533bf89-0807-479a-a90f-5bf4a456f1b2">

Additionally (think this is related? If not I can make a separate ticket) -- when I search for a druid I've added (to the top of the list of existing druids in Item edit view, ensuring I didn't delete any existing druids) the 6 or so druids I added aren't showing up in collection search, or in the list of items in the edit view [here](https://exhibits.stanford.edu/laurens-vt-test/catalog/admin?sort=timestamp).

**Comment by corylown** [comment]
@laurensorensen I think there might be some invalid geographic metadata in some of these records. I haven't identified all of them, but https://purl.stanford.edu/bc936sx1426.xml is an example.

```
    <extension displayLabel="geo">
      <rdf:RDF xmlns:gml="http://www.opengis.net/gml/3.2/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:gmd="http://www.isotc211.org/2005/gmd">
        <rdf:Description rdf:about="http://purl.stanford.edu/bc936sx1426">
          <dc:format>application/pdf</dc:format>
          <gmd:centerPoint>
            <gml:Point>
              <gml:pos>48°15'25.65"N 14°30'0.49"E</gml:pos>
            </gml:Point>
          </gmd:centerPoint>
        </rdf:Description>
      </rdf:RDF>
    </extension>
```

Related honeybadger error link: https://app.honeybadger.io/projects/49092/faults/99659726

```
RSolr::Error::Http: RSolr::Error::Http - 400 Bad Request Error: Async exception during distributed update: Error from server at http://sul-solr-prod-f.stanford.edu:8080/solr/exhibits-prod_shard1_replica_n19/: Bad Request request: http://sul-solr-prod-f.stanford.edu:8080/solr/exhibits-prod_shard1_replica_n19/ Remote error message: ERROR: [doc=bb392bt9034] Error adding field 'geographic_srpt'='[10°44'55.22"E 51°32'7.32"N]' msg=Unable to parse shape given formats "lat,lon", "x y" or as WKT because java.text.ParseException: Unknown Shape definition [10°44'55.22"E 51°32'7.32"N] 
```

**Comment by laurensorensen** [comment]
Thank you! Is it possible to know what inputted data is causing the error? Or I guess I could ask Arcadia?

**Comment by corylown** [comment]
I think the value here is not valid `<gml:pos>48°15'25.65"N 14°30'0.49"E</gml:pos>`

**Comment by laurensorensen** [comment]
Thanks!!

**Comment by jcoyne** [comment]
@corylown I think https://github.com/projectblacklight/spotlight/pull/2977 may also be needed

**Comment by corylown** [comment]
@laurensorensen it looks like the exhibits issue you reported via feedback last night for the `lauren-s-sandbox` exhibit is the same. This object has invalid geographic metadata that Solr can't index: https://purl.stanford.edu/bd283qd5772.xml

```
<extension displayLabel="geo">
  <rdf:RDF>
    <rdf:Description rdf:about="http://purl.stanford.edu/bd283qd5772">
      <dc:format>application/pdf</dc:format>
        <gmd:centerPoint>
          <gml:Point>
           <gml:pos>48°14'25.43"N 12°27'9.35"E</gml:pos>
         </gml:Point>
      </gmd:centerPoint>
    </rdf:Description>
  </rdf:RDF>
</extension>
```

**Comment by jcoyne** [comment]
`gml:pos` should look like this:
```
<gml:pos>40.689167 -74.044444</gml:pos>
```

See http://opengeospatial.github.io/e-learning/gml/text/main.html

**Comment by laurensorensen** [comment]
I'm seeing another indexing issue, but I moved to a different test exhibit. Should I add a new Github ticket or describe the issue here?

**Comment by laurensorensen** [comment]
Hmmm, that's really strange! Because bd283qd5772 is in a whole different exhibit/collection... European theater - which I didn't add any druids from to the Sandbox exhibit. These are the only four items in the sandbox exhibit:
zg893jk3553
xw889fc2359
dd739qy0841
fw846pf3883

**Comment by laurensorensen** [comment]
I uploaded these druids above as a test, and was going to change to the correct format for European and Pacific theater after successfully testing. Kim Durante helped me out and converted all the degree format latitude/longitude to decimal format, so those are available. And Arcadia confirmed that the MODS is working when I uploaded the 4 druids above...

**Comment by corylown** [comment]
@laurensorensen sorry, I think I found the wrong log entry. But I did find this for `zg893jk3553`. I'll try to figure out why...but I'm in interviews most of the day.

Error: ERROR: [doc=zg893jk3553] Error adding field 'geographic_srpt'='[34.30000000 129.3000000]' msg=Unable to parse shape given formats "lat,lon", "x y" or as WKT because java.text.ParseException: Unknown Shape definition [34.30000000 129.3000000]

**Comment by laurensorensen** [comment]
That's ok, and thanks very much for looking into it when you have a chance! Maybe I'll try and add a hyphen to the data to match the formatting that Justin posted above to see how it goes.

**Comment by corylown** [comment]
@laurensorensen I think the latitude and longitude are reversed in record zg893jk3553. They should be in latitude, longitude order and 129 isn't a valid latitude. I think the value you want in zg893jk3553 is `<gml:pos>34.30000000 129.3000000</gml:pos>`

**Comment by laurensorensen** [comment]
Thanks for this! Just getting back to this. I switched the latitude/longitude and their respective labels in these four druids in that same Sandbox exhibit, but am still not seeing a map on the item's exhibit page. The box 'inline map' has been checked in the metadata configuration... should I make a separate ticket? I cleared my cache (but not cookies). 

zg893jk3553
xw889fc2359
dd739qy0841
fw846pf3883
